### ISSUE TITLE
dataset.setGCPs() returns 0 as output coordinates

### ISSUE THREAD

**Issue by vmarquar** [naturalatlas/node-gdal#242]
I want to georeference a raster dataset with the help of ground control points, but i can't set them.

I tested it on a newly created dataset as well as on a copied dataset (using the `gdal.open()` and `driver.createCopy()`method  respectively).

My Ground Control Points (GCPs) are structured like this:
```
var gcp1 = {
	dfGCPLine: 2895.6639183,
	dfGCPPixel: 1812.12300713,
	dfGCPX: 4455000.0,
	dfGCPY: 5460000.0,
	dfGCPZ: 0
}
```
Then I run the `setGCP()` method and check the output:
```
ds.setGCPs([gcp1,gcp2,gcp3,gcp4],'EPSG:31468') // runs without errors
ds.getGCPs()
[ { pszId: '',
    pszInfo: '',
    dfGCPPixel: 0,
    dfGCPLine: 0,
    dfGCPX: 0,
    dfGCPY: 0,
    dfGCPZ: 0 },
  { pszId: '',
    pszInfo: '',
    dfGCPPixel: 0,
    dfGCPLine: 0,
    dfGCPX: 0,
    dfGCPY: 0,
    dfGCPZ: 0 },
  { pszId: '',
    pszInfo: '',
    dfGCPPixel: 0,
    dfGCPLine: 0,
    dfGCPX: 0,
    dfGCPY: 0,
    dfGCPZ: 0 },
  { pszId: '',
    pszInfo: '',
    dfGCPPixel: 0,
    dfGCPLine: 0,
    dfGCPX: 0,
    dfGCPY: 0,
    dfGCPZ: 0 } ]

```

I already tried a lot of combinations of input parameters for the projection option, e.g. 'EPSG:3857', converted well known text, numbers only and so on...

**Comment by sanjay900** [comment]
So i had a look, and have made a pull request to fix this issue.
https://github.com/naturalatlas/node-gdal/pull/243

**Comment by brandonreavis** [comment]
This is now fixed in version 0.9.8. Thanks @sanjay900 !
